acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketScmWorkflowJob.java

```diff
@@ -24,6 +27,11 @@ public BitbucketScmConfig bitbucketScmJenkinsFileSource() {
     }
 
     private void select(final String option) {
-        find(by.option(option)).click();
```

Comment:
This has been [observed](https://server-syd-bamboo.internal.atlassian.com/browse/BSERVTOOLS-BBSJEN-ST-835/test) to be flaky in the master branch as well as on my [branch](https://server-syd-bamboo.internal.atlassian.com/browse/BSERVTOOLS-BBSJEN509-22/commit).